template-compose/gradle/libs.versions.toml

```
@@ -0,0 +1,149 @@
+hilt-compiler = { group = "com.google.dagger", name = "hilt-compiler", version.ref = "hilt" }
+javax-inject = { group = "javax.inject", name = "javax.inject", version.ref = "javaxInject" }
+
+# Kotlin
+kotlin-coroutines-core = { group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-core", version.ref = "kotlinCoroutines" }
+
+# Log
+timber = { group = "com.jakewharton.timber", name = "timber", version.ref = "timber" }
+chucker = { group = "com.github.chuckerteam.chucker", name = "library", version.ref = "chucker" }
+chucker-no-op = { group = "com.github.chuckerteam.chucker", name = "library-no-op", version.ref = "chucker" }
+
+# Network
+moshi = { group = "com.squareup.moshi", name = "moshi", version.ref = "moshi" }
+moshi-adapters = { group = "com.squareup.moshi", name = "moshi-adapters", version.ref = "moshi" }
+moshi-kotlin = { group = "com.squareup.moshi", name = "moshi-kotlin", version.ref = "moshi" }
+okhttp = { group = "com.squareup.okhttp3", name = "okhttp", version.ref = "okhttp" }
+okhttp-logging = { group = "com.squareup.okhttp3", name = "logging-interceptor", version.ref = "okhttp" }
+retrofit = { group = "com.squareup.retrofit2", name = "retrofit", version.ref = "retrofit" }
+retrofit-moshi = { group = "com.squareup.retrofit2", name = "converter-moshi", version.ref = "retrofit" }
+
+# Util
+nimble-common = { group = "com.github.nimblehq", name = "android-common-ktx", version.ref = "nimbleCommon" }
+
+# Test
+test-core-ktx = { group = "androidx.test", name = "core-ktx", version.ref = "coreKtx" }
+test-junit = { group = "junit", name = "junit", version.ref = "junit" }
+test-kotlinx-coroutines-test = { group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-test", version.ref = "kotlinCoroutines" }
+test-kotest-assertions-core = { group = "io.kotest", name = "kotest-assertions-core", version.ref = "kotest" }
+test-mockk = { group = "io.mockk", name = "mockk", version.ref = "mockk" }
+test-turbine = { group = "app.cash.turbine", name = "turbine", version.ref = "turbine" }
+
+test-compose-ui = { group = "androidx.compose.ui", name = "ui-test-junit4" }
+test-robolectric = { group = "org.robolectric", name = "robolectric", version.ref = "robolectric" }
+
+[bundles]
+
+androidx = [
+    "androidx-core",
+    "androidx-lifecycle-runtime",
+    "androidx-lifecycle-compose",
+    "androidx-datastorePreferences",
+]
+
+compose = [
+    "compose-ui",
+    "compose-ui-tooling-preview",
+    "compose-material",
+    "compose-navigation",
+    "accompanist-permissions",
+]
+
+hilt = [
+    "hilt-android",
+    "hilt-navigation",
+]
+
+retrofit = [
+    "retrofit",
+    "retrofit-moshi",
+]
+
+okhttp = [
+    "okhttp",
+    "okhttp-logging",
+]
+
+moshi = [
+    "moshi-adapters",
+    "moshi-kotlin",
+]
+
+unitTest = [
+    "test-kotlinx-coroutines-test",
+    "test-junit",
+    "test-kotest-assertions-core",
+    "test-mockk",
+]
+
+implementationTest = [
+    "test-compose-ui",
+    "test-robolectric",
+]
```

Should we keep the convention to group them all in the same `bundle` group without separation, same as others versions, libs or plugins?

```suggestion
[bundles]
androidx = [
    "androidx-core",
    "androidx-lifecycle-runtime",
    "androidx-lifecycle-compose",
    "androidx-datastorePreferences",
]
compose = [
    "compose-ui",
    "compose-ui-tooling-preview",
    "compose-material",
    "compose-navigation",
    "accompanist-permissions",
]
hilt = [
    "hilt-android",
    "hilt-navigation",
]
retrofit = [
    "retrofit",
    "retrofit-moshi",
]
okhttp = [
    "okhttp",
    "okhttp-logging",
]
moshi = [
    "moshi-adapters",
    "moshi-kotlin",
]
unitTest = [
    "test-kotlinx-coroutines-test",
    "test-junit",
    "test-kotest-assertions-core",
    "test-mockk",
]
implementationTest = [
    "test-compose-ui",
    "test-robolectric",
]
```

---

Additionally, the `implementationTest` could be renamed to `uiTest` 🤔